spec/acceptance/self_service_spec.rb

```diff
@@ -149,6 +149,12 @@ class {'self_service':
         expect(result.stdout).to match(%r{false})
         run_shell('puppet config set use_cached_catalog false', expect_failures: false)
       end
+      it 'if S0040 conditions for false are met' do
+        run_shell('mv /etc/systemd/system/puppet_system_processes-metrics.timer /etc/systemd/system/puppet_system_processes-metrics.bak')
```

Comment:
run_shell('puppet agent --disable') i think your tests are failing sometimes due to puppet putting the timer back you may need to add a disable and enable into your rspec test